examples/vpc-scenario-1/main.tf

```diff
@@ -76,18 +80,24 @@ resource "aws_instance" "web" {
   key_name          = "${aws_key_pair.main.key_name}"
   instance_type     = "t2.nano"
   availability_zone = "${var.region}a"
+
   root_block_device {
     volume_type = "gp2"
     volume_size = "8"
   }
+
   associate_public_ip_address = "true"
-  vpc_security_group_ids      = ["${module.public-ssh-sg.id}",
-                                 "${module.open-egress-sg.id}"
+
+  vpc_security_group_ids = ["${module.public-ssh-sg.id}",
+    "${module.open-egress-sg.id}",
```

Comment:
This doens't seem like much of an improvement to me, but even so.. If the previous version were to be updated, I would have opted to put the `public-ssh-sg` module on a it's own line.